.github/workflows/playwright.yaml

```diff
@@ -31,6 +31,7 @@ jobs:
         'TestSettings:TEST_PASSWORD': '${{ secrets.PLAYWRIGHT_PASSWORD }}'
         'TestSettings:TEST_TOTP_SECRET': ${{ secrets.PLAYWRIGHT_TOTP_SECRET }}
         'TestSettings:TEST_BASE_URL': ${{ secrets.PLAYWRIGHT_BASE_URL }}
+        CI: 'true'
```

Comment:
is dit nu nog nodig?

Reply:
lijkt er op dat het nog goed gaat zonder deze property. weg laten dan maar?

Reply:
ja haal maar weg.

Reply:
hij is weg